HRCPP-474 Updated java test to 9.2.3

rigazilla opened the pull request:
Update java swig test to 9.2.3.
https://issues.jboss.org/browse/HRCPP-474

alanfx commented:
@rigazilla LGTM. Not sure how the `CrossLanguageHotRodTest` is exiting cleanly, but it is! Merging...

rigazilla commented:
Rebased and added some changes due to the new AdminOp features. Marked 2 AdminOp tests as failing: comments in the code.
The setup/cleanup things are now completely handled by the Java super classes architecture (SingleCacheManagerTest and ancestor), so no more cleanup is needed.

alanfx commented:
@rigazilla retested and ready to merge
